### ISSUE TITLE
DownStream (ASV: Automatic Speaker Verification) - missing information

### ISSUE THREAD

**Issue by OG-algo** [s3prl/s3prl#113]
Hi,
I would like to use the ASV with Mockingjay embedding (upstream). Reading the downstream doc, I have encountered a few questions without answer and I hope you can help me with them:

1. how can I use the pre-trained Mockingjay upstream with the ASV downstream?

2. how can I use my custom (.wav labeled) data to finetune this system?

3. what is the ASV architecture?
   3.1. what is the loss function?
   3.2. is there a paper describing this ASV task?

Thank you

**Comment by leo19941227** [comment]
Hi,

The downstream folder is majorly targeted on reproducing this paper:
[SUPERB: Speech processing Universal PERformance Benchmark](https://arxiv.org/abs/2105.01051)
You will find many information in it.

1. You can use `-u mockingjay` or `-u tera`, where the latter turns out to be better. But there are lots of other upstream model better than these two.
2. Currently the `sv_voxceleb1` folder is bound with VoxCeleb1 dataset. As using different dataset will require different definition of the torch dataset to correctly parsing the dataset structure. So this will not be an easy fix.
3.1. We use vanilla x-vector as the architecture since the downstream system is intended to be simple to effectively benchmark SSL representations. Loss function is Am-Softmax.
